lib/pat.c

```diff
@@ -3329,7 +3339,11 @@ grn_pat_fuzzy_search(grn_ctx *ctx,
     id = grn_pat_fuzzy_search_find_prefixed_start_node_id(ctx, &data);
   } else {
     PAT_AT(pat, GRN_ID_NIL, node);
-    id = node->lr[1];
+    if (node) {
+      id = pat_node_get_right(pat, node);
+    } else {
+      return GRN_NO_MEMORY_AVAILABLE;
```

Comment:
Can we use `ctx->rc`?

Reply:
Ah.Yes, we can.
Thank you.

I've fixed it.